soccer/gameplay/tactics/placement.py

```diff
@@ -0,0 +1,71 @@
+import single_robot_composite_behavior
+import behavior
+import main
+import role_assignment
+import enum
+import robocup
+import skills
+import constants
+import skills.pivot_kick
+import skills.capture
+import skills.move
+
+class Placement(single_robot_composite_behavior.SingleRobotCompositeBehavior):
+    class State(enum.Enum):
+        capturing = 1 #waiting
+        placing = 2 #setup
+        standby = 3 #ready
+
+    def __init__(self):
+        super().__init__(continuous=False)
+
+        for substate in Placement.State:
+            self.add_state(substate, behavior.Behavior.State.running)
+
+        self.add_transition(behavior.Behavior.State.start,
+            Placement.State.capturing, lambda: True, 'immediately')
+
+        self.add_transition(Placement.State.capturing, Placement.State.placing,
+            lambda: self.subbehavior_with_name('capture').state==behavior.Behavior.State.completed,
+            'done capturing')
+
+        self.add_transition(Placement.State.placing, Placement.State.standby,
+            lambda: self.subbehavior_with_name('placing').state==behavior.Behavior.State.completed,
+            'done placing')
+
+        self.add_transition(
+            Placement.State.standby, behavior.Behavior.State.completed,
+            lambda: self.subbehavior_with_name('standby').state == behavior.Behavior.State.completed,
```

Comment:
Not sure about this, but I think there needs to be something here about turning off the intake roller. Also we might want to make another move command to move away from the ball w/o touching it, before moving to the standby location. (You could just add the chosen robot to the line_up play for that, possibly).